index.js

```diff
@@ -18,81 +19,115 @@ function Fileupload(name, options) {
 	Collection.apply(this, arguments);
 	
 	// check to see if config has everything we need...
-	if (!this.config.properties || !this.config.directory || !this.config.fullDirectory) {
+	// We will no longer need a config.properties
+	// This 'should' be first run only, and shouldn't need to be saved...
+	if (!this.config.directory || !this.config.fullDirectory) {
```

Comment:
will be removing this part all together shortly... config will be set with this.config.blah || process.env.blah || "default"... saving the need for a automatic forced save